Healthy/Classes/Modules/Splash/SplashViewController.xib

```diff
@@ -0,0 +1,101 @@
+                <constraint firstItem="ccU-so-Lsr" firstAttribute="top" secondItem="kVU-ea-rFA" secondAttribute="bottom" constant="14" id="EHX-39-twH"/>
+                <constraint firstAttribute="bottom" secondItem="teD-mQ-e7K" secondAttribute="bottom" constant="96" id="Ivn-hp-bWN"/>
+                <constraint firstItem="EcO-QI-U0T" firstAttribute="leading" secondItem="fnl-2z-Ty3" secondAttribute="leading" constant="50" id="LLd-ck-soI"/>
+                <constraint firstItem="teD-mQ-e7K" firstAttribute="leading" secondItem="fnl-2z-Ty3" secondAttribute="leading" constant="66" id="RlJ-th-2Xx"/>
+                <constraint firstItem="fnl-2z-Ty3" firstAttribute="trailing" secondItem="mJ0-ee-dr3" secondAttribute="trailing" id="Srg-Rn-3M5"/>
+                <constraint firstItem="nw5-Nc-SnG" firstAttribute="top" secondItem="EcO-QI-U0T" secondAttribute="bottom" constant="20" id="TQ8-S3-tyT"/>
+                <constraint firstItem="fnl-2z-Ty3" firstAttribute="trailing" secondItem="ccU-so-Lsr" secondAttribute="trailing" constant="12" id="UjS-m9-m2J"/>
+                <constraint firstItem="kVU-ea-rFA" firstAttribute="centerX" secondItem="fnl-2z-Ty3" secondAttribute="centerX" id="XgR-AX-M6l"/>
+                <constraint firstItem="mJ0-ee-dr3" firstAttribute="width" secondItem="i5M-Pr-FkT" secondAttribute="width" id="b8N-WT-tI9"/>
+                <constraint firstItem="fnl-2z-Ty3" firstAttribute="trailing" secondItem="teD-mQ-e7K" secondAttribute="trailing" constant="66" id="dr8-PL-jlC"/>
+                <constraint firstItem="ccU-so-Lsr" firstAttribute="leading" secondItem="fnl-2z-Ty3" secondAttribute="leading" constant="12" id="heM-aV-UTH"/>
+                <constraint firstItem="kVU-ea-rFA" firstAttribute="top" secondItem="fnl-2z-Ty3" secondAttribute="top" constant="104" id="iOx-GR-PJn"/>
+                <constraint firstAttribute="bottom" secondItem="mJ0-ee-dr3" secondAttribute="bottomMargin" id="kmb-mT-qqF"/>
+                <constraint firstItem="nw5-Nc-SnG" firstAttribute="leading" secondItem="fnl-2z-Ty3" secondAttribute="leading" constant="50" id="qOu-3N-cqB"/>
+                <constraint firstItem="ccU-so-Lsr" firstAttribute="centerX" secondItem="kVU-ea-rFA" secondAttribute="centerX" id="vO7-lV-TFa"/>
+                <constraint firstItem="mJ0-ee-dr3" firstAttribute="top" secondItem="i5M-Pr-FkT" secondAttribute="top" id="yXO-XT-a4v"/>
+            </constraints>
+            <point key="canvasLocation" x="23.664122137404579" y="20.422535211267608"/>
+        </view>
+    </objects>
+    <resources>
+        <image name="logo-img" width="79" height="79"/>
+        <image name="splash-screen" width="375" height="812"/>
+        <systemColor name="systemBackgroundColor">
+            <color white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+        </systemColor>
+    </resources>
+</document>
```

Comment:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="21507" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
    <device id="retina6_12" orientation="portrait" appearance="light"/>
    <dependencies>
        <deployment identifier="iOS"/>
        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="21505"/>
        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
        <capability name="System colors in document resources" minToolsVersion="11.0"/>
        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
    </dependencies>
    <objects>
        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner" customClass="SplashViewController" customModule="Healthy" customModuleProvider="target">
            <connections>
                <outlet property="headerCaptionLabel" destination="nw5-Nc-SnG" id="Z3W-qH-rD0"/>
                <outlet property="headerTitleLabel" destination="EcO-QI-U0T" id="6ot-oB-7Fi"/>
                <outlet property="logoCaptionLabel" destination="ccU-so-Lsr" id="vkA-wp-DUV"/>
                <outlet property="startCookingButton" destination="teD-mQ-e7K" id="bdg-8R-b6J"/>
                <outlet property="view" destination="i5M-Pr-FkT" id="sfx-zR-JGt"/>
            </connections>
        </placeholder>
        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
        <view clearsContextBeforeDrawing="NO" contentMode="scaleToFill" id="i5M-Pr-FkT">
            <rect key="frame" x="0.0" y="0.0" width="393" height="852"/>
            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
            <subviews>
                <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFill" horizontalHuggingPriority="251" verticalHuggingPriority="251" preservesSuperviewLayoutMargins="YES" image="splash-screen" translatesAutoresizingMaskIntoConstraints="NO" id="mJ0-ee-dr3">
                    <rect key="frame" x="0.0" y="0.0" width="393" height="852"/>
                </imageView>
                <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" alignment="center" spacing="12" translatesAutoresizingMaskIntoConstraints="NO" id="zfe-CV-B0P">
                    <rect key="frame" x="36" y="155" width="321" height="111.33333333333331"/>
                    <subviews>
                        <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="logo-img" translatesAutoresizingMaskIntoConstraints="NO" id="kVU-ea-rFA">
                            <rect key="frame" x="121" y="0.0" width="79" height="79"/>
                            <constraints>
                                <constraint firstAttribute="width" constant="79" id="Bz1-5c-uR8"/>
                                <constraint firstAttribute="height" constant="79" id="zLL-BG-J83"/>
                            </constraints>
                            <edgeInsets key="layoutMargins" top="8" left="8" bottom="8" right="8"/>
                        </imageView>
                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="100K+ Premium Recipe " textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="ccU-so-Lsr" userLabel="100K+ Premium Recipe">
                            <rect key="frame" x="70.666666666666671" y="91" width="179.66666666666663" height="20.333333333333329"/>
                            <fontDescription key="fontDescription" type="system" pointSize="17"/>
                            <color key="textColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
                            <nil key="highlightedColor"/>
                        </label>
                    </subviews>
                </stackView>
                <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" alignment="center" spacing="80" translatesAutoresizingMaskIntoConstraints="NO" id="ueA-NA-GYL">
                    <rect key="frame" x="20" y="563.33333333333337" width="353" height="192.66666666666663"/>
                    <subviews>
                        <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" alignment="center" spacing="18" translatesAutoresizingMaskIntoConstraints="NO" id="wIb-w0-RlP">
                            <rect key="frame" x="56.666666666666686" y="0.0" width="240" height="58.666666666666664"/>
                            <subviews>
                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Get Cooking" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="EcO-QI-U0T">
                                    <rect key="frame" x="72.666666666666657" y="0.0" width="94.666666666666657" height="20.333333333333332"/>
                                    <fontDescription key="fontDescription" type="system" pointSize="17"/>
                                    <color key="textColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
                                    <nil key="highlightedColor"/>
                                </label>
                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Simple way to find Tasty Recipe" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="nw5-Nc-SnG">
                                    <rect key="frame" x="0.0" y="38.333333333333258" width="240" height="20.333333333333329"/>
                                    <fontDescription key="fontDescription" type="system" pointSize="17"/>
                                    <color key="textColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
                                    <nil key="highlightedColor"/>
                                </label>
                            </subviews>
                        </stackView>
                        <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="teD-mQ-e7K">
                            <rect key="frame" x="112.33333333333336" y="138.66666666666663" width="128.66666666666663" height="54"/>
                            <constraints>
                                <constraint firstAttribute="height" constant="54" id="QFs-F7-QyR"/>
                            </constraints>
                            <state key="normal" title="Button"/>
                            <buttonConfiguration key="configuration" style="plain" title="Start Cooking"/>
                            <connections>
                                <action selector="didTapStartCooking:" destination="-1" eventType="touchUpInside" id="21R-xk-30S"/>
                            </connections>
                        </button>
                    </subviews>
                </stackView>
            </subviews>
            <viewLayoutGuide key="safeArea" id="fnl-2z-Ty3"/>
            <color key="backgroundColor" systemColor="systemBackgroundColor"/>
            <constraints>
                <constraint firstItem="mJ0-ee-dr3" firstAttribute="leading" secondItem="fnl-2z-Ty3" secondAttribute="leading" id="49s-f7-KCr"/>
                <constraint firstAttribute="trailingMargin" secondItem="zfe-CV-B0P" secondAttribute="trailing" constant="20" id="7ag-f2-QhG"/>
                <constraint firstItem="ueA-NA-GYL" firstAttribute="leading" secondItem="fnl-2z-Ty3" secondAttribute="leading" constant="20" id="9gn-Z0-aWX"/>
                <constraint firstItem="zfe-CV-B0P" firstAttribute="leading" secondItem="i5M-Pr-FkT" secondAttribute="leadingMargin" constant="20" id="BUX-Yw-0vk"/>
                <constraint firstItem="zfe-CV-B0P" firstAttribute="top" secondItem="i5M-Pr-FkT" secondAttribute="topMargin" constant="96" id="Ez6-5D-cst"/>
                <constraint firstItem="fnl-2z-Ty3" firstAttribute="trailing" secondItem="mJ0-ee-dr3" secondAttribute="trailing" id="Srg-Rn-3M5"/>
                <constraint firstItem="mJ0-ee-dr3" firstAttribute="width" secondItem="i5M-Pr-FkT" secondAttribute="width" id="b8N-WT-tI9"/>
                <constraint firstAttribute="bottom" secondItem="mJ0-ee-dr3" secondAttribute="bottom" id="kmb-mT-qqF"/>
                <constraint firstItem="fnl-2z-Ty3" firstAttribute="trailing" secondItem="ueA-NA-GYL" secondAttribute="trailing" constant="20" id="l8V-HA-Tgs"/>
                <constraint firstAttribute="bottom" secondItem="ueA-NA-GYL" secondAttribute="bottom" constant="96" id="ujQ-ee-j2s"/>
                <constraint firstItem="mJ0-ee-dr3" firstAttribute="top" secondItem="i5M-Pr-FkT" secondAttribute="top" id="yXO-XT-a4v"/>
            </constraints>
            <point key="canvasLocation" x="23.664122137404579" y="20.422535211267608"/>
        </view>
    </objects>
    <resources>
        <image name="logo-img" width="79" height="79"/>
        <image name="splash-screen" width="375" height="812"/>
        <systemColor name="systemBackgroundColor">
            <color white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
        </systemColor>
    </resources>
</document>
```

Please check this version and let's discuss if you have any comment.

Reply:
<img width="369" alt="Screenshot 2023-05-27 at 7 26 52 PM" src="https://github.com/motoon-eg/healthy/assets/29166033/361ece3b-c70d-4574-9d60-b2cb28e8e250">
the margins look like this